lightning-network-book.tex

```diff
@@ -312,58 +312,58 @@ \chapter{Introduction}
+One core principle of this book is that we introduce new concepts by formulating problem statements first. We will then try to understand why these problems existed and how the technologies in this book will be able to solve them. In many cases, we will derive more specific problems which we will tackle in a similar fashion.
 
 \chapter{Bitcoin}
 \section{Introduction to Electronic Cash}
 \problem{How can we create a digital form of cash?}
-With the invention of the micro controller which lead to the development of computers, the Internet and smart phones, we have seen that many things that existed physically now exist in a digital version too.
+With the invention of the micro controller which lead to the development of computers, the Internet and smart phones, we have seen many things that existed physically now exist in a digital version too.
 Examples could be:
 \begin{itemize}
-\item Photographs, video recordings and music tapes.
-\item Books, newspapers and other print publications.
-\item Postal services for sending mail.
-\item Archives and libraries.
+\item Photographs, video recordings and music tapes
+\item Books, newspapers and other print publications
+\item Postal services for sending mail
+\item Archives and libraries
 \end{itemize}
 
 Also, a lot of communication has become digitized with services like email or instant messaging.
 It seems that a huge reason for society to adopt a technological breakthrough is that in many cases digital goods are more convenient than their physical counterparts.
-Take paper publications as an example: While it might be very convenient to carry around one book, it is hardly practical to carry around your entire library.
+Take paper publications as an example: While it might be very convenient to carry around one book, it is impractical to carry around your entire library.
 With modern electronic readers, it is possible to access the world's most extensive collection of textbooks on a single device.
 One can even easily store the entire Wikipedia --- which as an encyclopedia is larger than any other encyclopedia that has previously existed --- on a small nano SD card.
 Remember such an SD card is smaller than your thumbnail.
 
 With regard to cash, we can see a similar trend.
 Due to its scarcity gold served as one of the first universally accepted materials to serve as a currency.
-However, using gold was not very practical as it is a heavy material and cannot be split easily.
-Physical coins and bills (which in the beginning were backed by gold) emerged.
-This form of cash is still used in all major countries of the world.\footnote{although currency nowadays is no longer backed by gold.}
```

Comment:
why removing this footnote? isn't that true?

Reply:
I thought saying 'initially backed by gold' (or something to that effect) would be enough to imply that they're no longer backed by gold. Perhaps it's better to be more explicit though and keep the footer.